testutil/sims/app_helpers.go

```diff
@@ -47,6 +47,9 @@ var DefaultConsensusParams = &cmtproto.ConsensusParams{
 			cmttypes.ABCIPubKeyTypeEd25519,
 		},
 	},
+	Abci: &cmtproto.ABCIParams{
```

Comment:
Curious to learn about what made you add this change

Reply:
Sure. It's because of [this line](https://github.com/0xPolygon/heimdall-v2/pull/60/files#diff-ab0788124794a3e784e8940687917165b87afda23b94b047e08f5af14f60f38cR111) we use for initializing our heimdall test application. 
Without this change `VoteExtensionsEnableHeight` is `nil` and `mustAddSpecialTransaction` function was failing because of this
```
	if height <= ctx.ConsensusParams().Abci.VoteExtensionsEnableHeight
```

Reply:
Sounds good, thanks. I did it [here](https://github.com/0xPolygon/heimdall-v2/pull/60/commits/efd13938712355878c98064dba819578efa3d948).
Nevertheless, I believe such change in our fork of `cosmos` 
```
Abci: &cmtproto.ABCIParams{
		VoteExtensionsEnableHeight: 1,
	},
```
would still be needed. In this way, every time we leverage the `testutils` in `v2` we get the vote extensions enabled "for free", which is not the default case for `simapp` here.

Reply:
Good point!